Description: better quit
 update key to the new SDL2 layout
 and even add more options to quit the app
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Last-Update: 2020-04-22
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/main.c
+++ b/main.c
@@ -805,15 +805,13 @@
           switch( event.key.keysym.sym )
           {
 #ifndef __FULLSCREEN_ONLY__
-            case 'f':
-            case 'F':
+            case SDLK_f:
             case SDLK_RETURN:	// was going to use alt-return, but since return isn't doing anything else..
               SDL_SetWindowFullscreen( sdlWindow,1 );
               break;
 #endif
 
-            case 'p':
-            case 'P':
+            case SDLK_p:
               paused ^= 1;
               SDL_PauseAudio( paused );
               break;
@@ -834,7 +832,11 @@
 //              else
               done = 1;
               break;
-            
+
+	    case SDLK_q:
+              done = 1;
+              break;
+
             default:
               break;
           }
